microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -736,13 +722,30 @@ This is a programmatic way to record JavaScript execution, which can be useful i
 When DevTools is used to debug processes that are running in Node.js, this experiment makes whatever `console.profile` generates available to the **Performance** Tool.
 
 Status:
-*  This checkbox is present in Microsoft Edge Canary 111.
-*  This checkbox is not present in Microsoft Edge Stable 108.
+*  This checkbox is present in Microsoft Edge Canary 112.
+*  This checkbox is present in Microsoft Edge Stable 109.
 
 See also:
 * [JavaScript Profiler tool](../javascript-profiler/javascript-profiler-tool.md)
 
 
+<!-- ====================================================================== -->
+## View system nodes of CPUProfile in the Performance panel
+
+_new as of Microsoft Edge 112_
+
+Whether to display system nodes of CPUProfile in the **Performance** tool.
+<!-- todo: need text -->
```

Comment:
@captainbrosset @zoherghadyali - What does the **View system nodes of CPUProfile in the Performance panel** checkbox experiment do?  

Where to link to?